### DISCUSSION TITLE
Reinstalled W11, OpenShell Not Fully Right

### DISCUSSION THREAD

**Discussion by LookingForWayOut** [Open-Shell/Open-Shell-Menu#1544] [Q&A] [answered]
Hello.
Before today, for a year, OpenShell has been fine on my W11 Pro. Today I was forced to wipe and reload my desktop because of a blasted Killer NIC bug with M$. I installed 4.4.130 once my Windows was up and updated. When the install was done, it did not popup with the usual new install config window. Doing so manually, it would not even take my backup settings I exported. And I can not make it work like I did before, when the Start icon is the default one of Windows instead of the shell icon.
Any thoughts about any of this?
Thank you.

**Comment by tallman69** [comment, marked as answer]
I would try using v4.4.189.  It is a prerelease but it has been working well on my two Windows 11 22H2 machines.  I also have it installed on my Windows 10 machine without any issues.  If you use an older version of OpenShell on Windows 11 you would most likely need to also run ExplorerPatcher for the older versions of OpenShell to work well.

**Comment by LookingForWayOut** [comment]
Ah, that helped.
I used the latest version of ExplorerPatcher 22621.1555.55.1. Once installed, I ran OpenShell install (still 4.4.130 but would have gone lower as you suggested if this attempt failed) as administrator (just in case). And tubularly bodacious, it worked. I got my Classic Style and stock Window$ Start icon (the four squares, the OCD of me).
Thank you!

**Reply by tallman69** [reply]
You're welcome!  Glad it has all worked out.
